packages/web-components/src/components/va-text-input/va-text-input.scss

```diff
@@ -24,15 +24,6 @@
   font-family: var(--font-source-sans);
 }
 
-:host(:focus) {
-  outline: none !important;
-}
-
-:host(:not([uswds='false'])) input:not([disabled]):focus {
-  outline: 2px solid var(--vads-color-action-focus-on-light);
-  outline-offset: 2px;
```

Comment:
This is redundant because the [focusable mixin](https://github.com/department-of-veterans-affairs/component-library/blob/7810b0123ddb9dee58d491ce0ebd1bb8fc3db072/packages/web-components/src/mixins/focusable.css) imported at the top of the file is doing the same thing.